service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -51,6 +51,10 @@ public boolean isCloseCaseEnabled() {
         return ldClient.boolVariation("close-case", createLDUser(), false);
     }
 
+    public boolean isNewCaseStateModelEnabled() {
+        return ldClient.boolVariation("new-case-state-model", createLDUser(), false);
+    }
```

Comment:
are we not calling this 'issue-resolution-state' because it's going to work as the toggle for all our new case states?

Reply:
Exactly. It will cover 'Final hearing' state also.

Reply:
cool - makes sense then. :)